### PR TITLE
Update Concept and Scheme models

### LINKED ISSUE #123: Re-migrate concept->reference nodes with default values
- [x] capture default values configured for concept nodes now that reference datatype supports default values
- [x] update modeling issues to be enumerated here:
  - [x] Nodegroups with string content node and corresponding language node should use the non-localized string datatype for the content/label/statement node and a single-select reference datatype for language node:
    - [x] Scheme:
      - [x] right_statement_content + right_statement_language
      - [x] appellative_status_ascribed_name_content + appellative_status_ascribed_name_language
      - [x] statement_content_n1 + statement_language_n1
    - [x] Concept:
      - [x] statement_content + statement_language
      - [x] statement_data_assignment_statement_content + statement_data_assignment_statement_language
      - [x] appellative_status_ascribed_name_content + appellative_status_ascribed_name_language
      - [x] data_assignment_statement_content  + data_assignment_language
  - [x] Concepts should only participate in one scheme - Part of Scheme and Top Concept Of should be cardinality-1 and a single-select resource instance nodes

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
Resolves #123 

This will break the test data - the RDM -> Lingo migration will need to be re-run, and likely updated based on new model configurations.

### CODE REVIEW

**Review by robgaston — APPROVED** [review]
lgtm! thanks!

**Comment by johnatawnclementawn** [comment]
1. load [AFS thesaurus](https://github.com/archesproject/arches-for-science/blob/dev/2.0.x/arches_for_science/pkg/reference_data/concepts/disco_aat_thesaurus.xml)
2. start celery (I use `python -m celery -A arches_lingo worker -l info`)
3. Run RDM to Lingo ETL module for `AAT Entries` http://localhost:8000/plugins/bulk-data-manager
4. Verify number of part_of_scheme tiles created matches the number of concept resources created

File: arches_lingo/etl_modules/migrate_to_lingo.py [hunk 1]
```diff
@@ -442,10 +442,22 @@ def init_relationships(
...
             resourceinstanceid = concept["concept"]
 
+            concept_has_part_of_scheme = resourceinstanceid in concepts_with_scheme
+            existing_scheme = concepts_with_scheme.get(resourceinstanceid)
+            if not concept_has_part_of_scheme:
+                concepts_with_scheme[resourceinstanceid] = root_list
+            elif concept_has_part_of_scheme and existing_scheme == root_list:
+                continue
+            elif concept_has_part_of_scheme and existing_scheme != root_list:
+                raise Exception(
```
Comment (jacobtylerwalls) [inline arches_lingo/etl_modules/migrate_to_lingo.py:457]:
```suggestion
                raise CommandError(
```

**Review by jacobtylerwalls — APPROVED** [review]
+1 tested. Few things noticed along the way that can be follow-ups:

- make etl_modules an explicit package, so that it's measured by coverage
- use CommandError
- catch CommandError and surface the error message in the GUI, but may want to check with @chiatt

**Comment by johnatawnclementawn** [comment]
@robgaston python coverage was previously passing because Lingo ETL Modules was not a package. Given the blocking nature of these model changes for other work in flight, I created a follow-up issue #131. Would you like to see that addressed in this PR or okay to do the follow up?

**Comment by robgaston** [comment]
@johnatawnclementawn it's fine to address the coverage in the follow up ticket

File: arches_lingo/etl_modules/migrate_to_lingo.py (hunk 1 above)
Comment (johnatawnclementawn) [reply arches_lingo/etl_modules/migrate_to_lingo.py]: Updated to follow pattern in [single_csv here](https://github.com/archesproject/arches/blob/ab1e5bf7038b304cdd10f93bd1f4a27829022549/arches/app/etl_modules/import_single_csv.py#L192C1-L198C14). Will need to think about how to handle errors like this when we update this ETL module to be run via CLI, re. https://github.com/archesproject/arches/pull/10519

**Pull request merged**